gril/grilrequest.h

```diff
@@ -205,13 +205,17 @@ void g_ril_request_set_smsc_address(GRil *gril,
 					struct parcel *rilp);
 
 void g_ril_request_dial(GRil *gril,
-			const struct ofono_phone_number *ph,
-			enum ofono_clir_option clir,
-			struct parcel *rilp);
+                      const struct ofono_phone_number *ph,
+                      gboolean isEmergency,
+                      enum ofono_clir_option clir,
+                      struct parcel *rilp);
+
```

Comment:
Seems like you inadvertently change the whitespace here. Remember, oFono uses tab characters, not spaces.